README.md

```diff
@@ -22,12 +21,19 @@ Add the following permissions to the `AndroidManifest.xml`:
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="de.julianassmann.flutter_background_example">
 
-    <!-- Adapt to the foreground service type desired -->
+    <!-- General permissions for the flutter_background plugin -->
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
+    <uses-permission android:name="android.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS" />
```

Comment:
These should not be necessary anymore. They are automatically copied over from the plugin manifest.

Reply:
Yes, I noticed it too and have fixed it in #91. Thanks for letting me know :)

Reply:
Excellent 👌

New release is greatly appreciated ❤️